extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -131,8 +132,14 @@ export class DoubleclickA4aEligibility {
           'be supported starting on March 29, 2018. Please refer to ' +
           'https://github.com/ampproject/amphtml/issues/11834 ' +
           'for more information');
+    let json;
```

Comment:
const usdrd = 'useSameDomainRenderingUntilDeprecated';
const hasUSDRD = (tryParseJson(element.getAttribute('json')) || {})[usdrd] || usdrd in element.dataset;
if (hasUSDRD) {
   warnDeprecation(usrdr);
}

Reply:
done